.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - '6'
+
+dist: trusty
+sudo: required
```

Comment:
Remebered one more reason: Postgres >= 9.5 is offered only on Trusty not on default ones. I tried also this option.